source/docs/articles/wordpress/importing-a-wordpress-site.md

```diff
@@ -6,53 +6,122 @@ category:
   - getting-started
 keywords: wordpress, importing, import site
 ---
-## Overview  
-<div class="alert alert-info" role="alert"> <strong>Note:</strong> Most WordPress sites with session-using code are relying on PHP's default session manager, which uses temporary files on local disk. Pantheon does not support this because it will not work properly in our distributed environment. You can read <a href="/docs/articles/wordpress/wordpress-and-php-sessions#wordpress-and-php-sessions">more here</a>.</div>
 
-##Prepare for Import  
-You will need a zip or tar.gz archive of your site in three separate files:
+There are three major components that make up a dynamic site:
 
-* Codebase - all executable code, plugins, themes, and so forth.
+1. **Codebase** - all executable code, plugins, themes, and so forth.
 
-* Database - contains the content of the site and some site configurations.
+2. **Database** - contains the content of the site and some site configurations.
 
-* Files - anything under wp-content/uploads. This houses a combination of uploaded content from site users, along with generated stylesheets, aggregated scripts, image styles, etc.
+3. **Files** - anything under `wp-content/uploads`. This houses a combination of uploaded content from site users, along with generated stylesheets, aggregated scripts, image styles, etc.
 
-There are two ways to import. Direct uploads from your desktop can be 100MB max; using a remote URL can be up to 500MB.
+<div class="alert alert-info" role="alert"> <strong>Note:</strong> Most WordPress sites with session-using code are relying on PHP's default session manager, which uses temporary files on local disk. Pantheon does not support this because it will not work properly in our distributed environment. You can read <a href="/docs/articles/wordpress/wordpress-and-php-sessions#wordpress-and-php-sessions">more here</a>.</div>
 
-Popular plugins like [Duplicator](http://wordpress.org/plugins/duplicator/) and [BackupBuddy](http://ithemes.com/codex/page/BackupBuddy) will also do this for you. You should be able to simply upload the archive file they produce and Pantheon will do the rest. For large imports, pasting in a web-readable url (e.g. dropbox secret link) will be much faster than manually uploading. Make sure a link goes directly to the file, not a landing page.
+## Export WordPress via Plugins
+Popular plugins like [Duplicator](/docs/articles/wordpress/clone-a-wordpress-site-with-duplicator-plugin/) and [BackupBuddy](http://ithemes.com/codex/page/BackupBuddy) allow you to create single file archives quickly. For large imports, pasting in a web-readable url (e.g. Dropbox secret link) will be much faster than manually uploading. Make sure a link goes directly to the file, not a landing page.

```

Comment:
link to our other docs?
https://pantheon.io/docs/articles/wordpress/clone-a-wordpress-site-with-duplicator-plugin/
https://pantheon.io/docs/articles/sites/create/importing-a-large-site/